### ISSUE TITLE
Grasp DB with IKEA objects

### ISSUE THREAD

**Issue by carlosjoserg** [CentroEPiaggio/pacman-DR53#5]
The current Grasp DB has too many errors in body tracking within a grasp sequence.
1. After CentroEPiaggio/phase-space#2 is resolved and Object Pose Estimation is finished by @Tabjones, acquire grasps with IKEA objects to fill [this dataset](https://github.com/CentroEPiaggio/unipi-grasp-datasets/tree/master/pacman_wp2_grasp_db).
2. Transfer acquisitions to the internal DB format (help wanted from @alessandrosettimi).

**Comment by hamalMarino** [comment]
At the current stage, there is no registered hand joint position: I would say that at least a few objects (first 5 as we agreed with @Tabjones) should have recorded grasp, then the rest will take some more time and be a matter of secondary importance.
About transferring to internal DB format, [this commit](https://bitbucket.org/dualmanipulation/grasp_db/commits/8b653ccdbff2b2d96d4fea2679f3f77c7fadab33) solves the issue: there is now a very simple utility (called grasp_storage) which allows this from command line with almost no effort (so, don't worry @alessandrosettimi !)

**Comment by carlosjoserg** [comment]
> At the current stage, there is no registered hand joint position...

What do you need that for? If it is to discriminate the approach/grasp phases, the flexiforce readings can give you a good estimate for that as well, as long as you record them of course, which seems to be the case [here](https://github.com/CentroEPiaggio/unipi-grasp-datasets/blob/master/pacman_wp2_grasp_db/data/startRecording.sh#L21)

> About transferring to internal DB format...

One question, the almost-no-effort is always that ctrl-c/ctrl-v of the timestamp from the bag play into the code, and then recompiling?

**Comment by hamalMarino** [comment]
Nope, have a look at the code if you want, but it's all command line...
Still, you need the timing written somewhere, even in a file will work (try
to rosrun dual_manipulation_grasp_db grasp_storage, I hope the help is
clear enough!)

On Monday, April 13, 2015, Carlos J. Rosales notifications@github.com
wrote:

> At the current stage, there is no registered hand joint position...
> 
> What do you need that for? If it is to discriminate the approach/grasp
> phases, the flexiforce readings can give you a good estimate for that as
> well, as long as you record them of course, which seems to be the case
> here
> https://github.com/CentroEPiaggio/unipi-grasp-datasets/blob/master/pacman_wp2_grasp_db/data/startRecording.sh#L21
> 
> About transferring to internal DB format...
> 
> One question, the almost-no-effort is always that ctrl-c/ctrl-v of the
> timestamp from the bag play into the code, and then recompiling?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/CentroEPiaggio/pacman-DR53/issues/5#issuecomment-92347599
> .

## 

---

Ing. _Hamal Marino_
PhD Student in Automation, Robotics, and Bioengineering

Research Center “E.Piaggio”
Faculty of Engineering - University of Pisa
Largo Lucio Lazzarino, 1
56122 Pisa - Italy
Tel. +39.050.2217050
Fax +39.050.2217051

Email: hamal.marino@centropiaggio.unipi.it

## http://www.linkedin.com/in/hamalmarino: linkedin.com/in/hamalmarino

**Comment by carlosjoserg** [comment]
Yes, it looks clear, thanks !
